website/docs/getting-started/for-designers.md

```diff
@@ -118,9 +118,9 @@ Open the [resources menu](https://help.figma.com/hc/en-us/articles/360039150413-
 
 #### Copying from the stickersheet
 
-If you prefer to select a component visually, the [Product Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?t=Ooe3pkDap3cGcgAH-1) library has a stickersheet for each component displaying the available variants and properties. Just like when using the direct inserts, this method maintains the link to the main component in the library, which means it will receive updates as they become available.
+If you prefer to select a component visually, the [Product Components](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67397-95918&t=gWdKy44MzTP4cTRo-1) library has a stickersheet for each component displaying the available variants and properties. Just like when using the direct inserts, this method maintains the link to the main component in the library, which means it will receive updates as they become available.
```

Comment:
This link leads to the Button stickersheet. If that is intentional should be have a note to let consumers know what to expect when clicking this link?

Reply:
It previously linked to the Button component, so I thought swapping it made the most sense since that stickersheet has has a pretty clear breakdown of variants. I'm not sure how essential it is to be overly explicit here as this is more of an example rather than specific steps.

Reply:
Got it, if that makes sense to you I'm fine with it. I don't have strong feelings on this one if it was that way before.

Reply:
What about clarifying where the link goes? Something like: 

If you prefer to select a component visually, the Product Components library has a stickersheet displaying each component's available variants and properties (see Button stickersheet).